api/GamepadEvent.json

```diff
@@ -189,9 +196,16 @@
             "safari_ios": {
               "version_added": "10.3"
             },
-            "samsunginternet_android": {
-              "version_added": true
-            },
+            "samsunginternet_android": [
+              {
+                "version_added": "3.0"
+              },
+              {
+                "prefix": "webkit",
```

Comment:
Same here, I don't trust the data being mirrored, this `gamepad` property probably wasn't prefixed.